src/hardware/mixer.cpp

```diff
@@ -461,12 +461,10 @@ void MixerChannel::AddSamples_s32_nonnative(Bitu len,const Bit32s * data) {
 }
 
 void MixerChannel::FillUp(void) {
```

Comment:
Remove `void` while we're at it ;)

Reply:
Took a pass through the mixer and cleaned these up.